spec/filter-spec.coffee

```diff
@@ -112,6 +112,9 @@ describe "filtering", ->
     expect(bestMatch(['z_a_b', 'a_b'], 'ab')).toBe 'a_b'
     expect(bestMatch(['a_b_c', 'c_a_b'], 'ab')).toBe 'a_b_c'
 
+  it "weighs matches that are substring matches higher", ->
+    expect(bestMatch(['/a/b/c/install.txt', 'inst-all.txt'])).toBe '/a/b/c/install.txt'
```

Comment:
All the other examples pass in an array of candidates and then a search string. All you're passing in here are candidates?